Bottom: 8e397ed05daebaa1f96e115cb41b5827c62504a7
Top:    19cbb1a735838bb5bd588caf9611f7bdd8c40a28
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-10 13:58:04 -0500

Add parameter for buildbot version

---
diff --git a/docker/buildbot-master/Dockerfile b/docker/buildbot-master/Dockerfile
index 521c823..dde9cdb 100644
--- a/docker/buildbot-master/Dockerfile
+++ b/docker/buildbot-master/Dockerfile
@@ -1,4 +1,5 @@
-FROM        buildbot/buildbot-master:v0.9.13
+ARG         BUILDBOT_VERSION=latest
+FROM        buildbot/buildbot-master:$BUILDBOT_VERSION
 MAINTAINER  Andrew Cooper <andrew.cooper@novatechweb.com>
 
 WORKDIR /buildbot
diff --git a/docker/buildbot-worker-ntel/Dockerfile b/docker/buildbot-worker-ntel/Dockerfile
index c85bf86..aa0e98e 100644
--- a/docker/buildbot-worker-ntel/Dockerfile
+++ b/docker/buildbot-worker-ntel/Dockerfile
@@ -58,12 +58,13 @@ RUN echo "dash dash/sh boolean false" \
 |   debconf-set-selections -v \
 &&  dpkg-reconfigure -f noninteractive dash
 
+ARG BUILDBOT_VERSION
 # Install required python packages, and twisted
 RUN wget https://bootstrap.pypa.io/get-pip.py \
 &&  python get-pip.py --no-cache-dir \
 &&  pip --no-cache-dir install \
     'twisted[tls]' \
-    'buildbot-worker==0.9.13'
+    buildbot-worker==${BUILDBOT_VERSION}
 
 # Install software for image signing
 RUN curl http://172.16.64.2/~georgem/610-009981-015_SW_PTK_5.3_Client_RevA.tar \
diff --git a/docker/buildbot-worker-ptxdist/Dockerfile b/docker/buildbot-worker-ptxdist/Dockerfile
index c29a39e..b5a102e 100644
--- a/docker/buildbot-worker-ptxdist/Dockerfile
+++ b/docker/buildbot-worker-ptxdist/Dockerfile
@@ -57,13 +57,14 @@ RUN DEBIAN_FRONTEND=noninteractive \
         yasm \
 &&  rm -rf /var/lib/apt/lists/*
 
+ARG BUILDBOT_VERSION
 # Install required python packages, and twisted
 RUN wget https://bootstrap.pypa.io/get-pip.py \
 &&  python get-pip.py --no-cache-dir \
 &&  pip --no-cache-dir install \
         'virtualenv' \
         'twisted[tls]' \
-        'buildbot-worker==0.9.13' \
+        buildbot-worker==${BUILDBOT_VERSION} \
 &&  rm -rf get-pip.py
 
 ARG BUILDBOT_UID=1000
